gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -189,13 +177,16 @@
                 <div class="col-md-6 form-group">
                   <label translate>Folder Type</label>
                   &nbsp;<a href="https://docs.syncthing.net/users/foldertypes.html" target="_blank"><span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span></a>
-                  <select class="form-control" ng-model="currentFolder.type">
+                  <select class="form-control" ng-model="currentFolder.type" ng-disabled="editingExisting && currentFolder.type === 'encrypted'">
                     <option value="sendreceive" translate>Send &amp; Receive</option>
                     <option value="sendonly" translate>Send Only</option>
                     <option value="receiveonly" translate>Receive Only</option>
+                    <option value="encrypted" ng-disabled="editingExisting" translate>Encrypted</option>
                   </select>
                   <p ng-if="currentFolder.type == 'sendonly'" translate class="help-block">Files are protected from changes made on other devices, but changes made on this device will be sent to the rest of the cluster.</p>
                   <p ng-if="currentFolder.type == 'receiveonly'" translate class="help-block">Files are synchronized from the cluster, but any changes made locally will not be sent to other devices.</p>
+                  <p ng-if="currentFolder.type == 'encrypted'" translate class="help-block">Receives only encrypted data, i.e. cannot read stored data. All remote devices need to be set up with the same password or be encrypted themselves to share this folder.</p>
+                  <p ng-if="editingExisting" translate class="help-block">Type cannot be changed from/to encrypted on an existing folder. You need to remove it, potentially clean up/decrypt data on disk, and re-add it.</p>
```

Comment:
Suggestion for easier reading/translation:
Folder type "encrypted" cannot be changed after saving. You need to remove the folder, optionally delete files on disk, and add it again.